server/repository/src/db_diesel/name_tag_row.rs

```diff
@@ -54,6 +54,14 @@ impl<'a> NameTagRowRepository<'a> {
         Ok(result)
     }
 
+    pub fn find_one_by_name(&self, name: &str) -> Result<Option<NameTagRow>, RepositoryError> {
+        let result = name_tag_dsl::name_tag
+            .filter(name_tag_dsl::name.eq(name))
```

Comment:
I think we should do a case insensitive match here...

![image](https://user-images.githubusercontent.com/8287373/231952646-a9765581-4de9-4a4c-871f-27c93e3d574d.png)

It looks like mSupply takes the first 'case' it finds and saves that, but then it doesn't create duplicates for different cases.